fixtures/cops.py

```diff
@@ -0,0 +1,82 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import pytest
+import requests
+
+import tldextract
+from rex_redirects import get_rex_release_json_url
+
+from tests.utils import gen_from_file, skip_if_destructive_and_sensitive
+
+DATA_DIR = os.path.join(os.path.realpath(os.path.dirname(__file__)), "data", "webview")
+
+__all__ = ["cops_base_url"]
+
+
+@pytest.fixture
+def cops_base_url(request):
+    """Return a base URL for cops"""
+    config = request.config
+    base_url = config.getoption("cops_base_url") or config.getini("cops_base_url")
+    if base_url is not None:
+        skip_if_destructive_and_sensitive(request, base_url)
```

Comment:
I may be wrong or misunderstanding how this works, but will this skipping ever get invoked? The `sensitive_url=^(?:https?://)?(?:archive\.|legacy\.)?cnx\.org` regex in `setup.cfg` will never match `https://cops.openstax.org/` for example. Is this something we want to address?

Reply:
That's a good point. I just copied this from qwebview and used for cops.